### PR TITLE
Fix basic setup test and add indexed property test

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by zcorpan** [comment]
In Chromium and Gecko the [[Delete]] test passes. In WebKit (STP, WebKit 11603.1.2) I get "Harness status: Timeout" and [[Delete]] test fails (but also the first "Basic sanity-checking test" fails). I had popup blocker disabled.

**Comment by annevk** [comment]
See the issues this was referenced from. I think that's being worked on.

File: html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html [hunk 1]
```diff
@@ -51,7 +51,10 @@
  */
 
 addTest(function() {
-  assert_equals(location.host, '{{domains[]}}:{{location[port]}}', 'Need to run the top-level test from {{domains[]}}:{{location[port]}}');
+  // Note: we do not check location.hostname as its default port semantics are hard to reflect
```
Comment (bzbarsky) [inline html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html]: You mean you don't check location.host, yes?

File: html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html [hunk 2]
```diff
@@ -189,6 +192,8 @@
  * [[Delete]]
  */
 addTest(function() {
+  assert_throws(null, function() { delete C[0]; }, "Can't delete cross-origin indexed property");
```
Comment (bzbarsky) [inline html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html]: Shouldn't all these tests be asserting something about the type of exception thrown?  Presumably the spec specifies that...

File: html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html (hunk 1 above)
Comment (annevk) [inline html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html]: 😟

File: html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html (hunk 2 above)
Comment (annevk) [inline html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html]: See https://github.com/whatwg/html/issues/1727 where I offered to do that. Felt like a separate change.

**Pull request merged**